scipy/stats/tests/test_mstats_basic.py

```diff
@@ -550,6 +550,20 @@ def test_trimmedmean(self):
         assert_almost_equal(mstats.trimmed_mean(data,(0.1,0.1)), 343, 0)
         assert_almost_equal(mstats.trimmed_mean(data,(0.2,0.2)), 283, 0)
 
+    def test_trimmedvar(self):
+        data = ma.array([77, 87, 88, 114, 151, 210, 219, 246, 253, 262,
+                         296, 299, 306, 376, 428, 515, 666, 1310, 2611])
+        assert_allclose(mstats.trimmed_var(data, 0.1), 79989.9723)
```

Comment:
What is the source of this reference value? Either the source of the value should be noted, or - better yet - the reference value can be calculated in the test using an approach that is somewhat different from the implementation in the PR (e.g. manually eliminate the elements to be trimmed, then use `np.var` to calculate the variance).

Reply:
This reference value was obtained by running the function itself, I thought the implementation is correct enough to give the correct output.
>reference value can be calculated in the test using an approach that is somewhat different from the implementation in the PR

Right, I will do that.

Reply:
I have done self implementation, please check.